android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/Image.kt

```diff
@@ -183,8 +182,8 @@ fun ClickableImageIcon(
                 .fillMaxSize(0.9f),
             bitmap = imageConfig.decodedBitmap!!.asImageBitmap(),
             contentDescription = null,
-            contentScale = ContentScale.Crop,
-            colorFilter = ColorFilter.tint(tint ?: imageProperties.imageConfig?.color.parseColor()),
+            alpha = 1.0f,
+            contentScale = ContentScale.Fit,
```

Comment:
Is there any reason for this change?

Reply:
Using ` ContentScale.Fit ` here ensures the image maintains its original aspect ratio. This prevents distortion and guarantees the entire image is visible, even if there's some empty space around it. useful for where preserving the complete image and its proportions is crucial

Setting` alpha = 1.0f ` here ensures the element maintains full opacity. This means it will be completely visible and not transparent at all

Reply:
Can we make both configurable?

Reply:
For the `alpha` default to 1.0f

Reply:
noted @ellykits